### ISSUE TITLE
[BUG] Shizuku Installer doesn't work

### ISSUE THREAD

**Issue by JFMugen** [Droid-ify/client#781]
**Describe the bug**
When I try to install something with Shizuku it just creates an infinite "Installing" loop and I can't click on cancel. 

**To Reproduce**
1- Enable Shizuku from its app, it is working. Tried with another app as well to be sure (Canta), works as intended. 
2- Open Droidify, select "Shizuku Installer"
3- Try to install any app and it stuck at "Installing", forces me to kill the app. 

**Expected behavior**
It works. 

**Smartphone (please complete the following information):**
 - Device: Xiaomi Redmi 12
 - OS: Android 14
 - Version 1.0.6.0 
 - Everything is updated to latest

**Additional context**
Feel free to ask anything else.

**Comment by JFMugen** [comment]
Alright... I am not sure if this is how it is supposed to be but when I click "install via USB" it works. Problem is now I have to "Allow" security prompt "do you want to install X app via USB" every time. Is it possible to bypass that for a total background installation?

**Comment by KaranKad** [comment]
I have same issue on Poco F6, its due to system optimization settings in developer options for hyperos.
I tried various app installers with shizuku such as [Aurora Store](https://gitlab.com/AuroraOSS/AuroraStore) and [SAI](https://github.com/Aefyr/SAI) and they all reported with same error `INSTALL_FAILED_USER_RESTRICTED: Install canceled by user`. 
System optimization setting is borking down adb permission on hyperos. Turning it off fixes this issue but creates more issues on the system. It messed up app icons, device felt sluggish and for some reason removed screen recorder.

**Comment by Iamlooker** [comment]
@JFMugen if @KaranKad fixed the issue please close the issue

**Comment by AzzyC** [comment]
For me to get apps to install with Shizuku Installer method, I had to enable installing apps via USB which I suppose is the intended function. Have you enabled this toggle in developer options on your device?

**Comment by KaranKad** [comment]
> For me to get apps to install with Shizuku Installer method, I had to enable installing apps via USB which I suppose is the intended function. Have you enabled this toggle in developer options on your device?

Yeah it works but you need to enable installing apps via USB for all the apps going onwards, which defeats the purpose of using shizuku to install apps seamlessly.

**Comment by Iamlooker** [comment]
@KaranKad can you try legacy installer by disabling developer options, check if it gives any error

If it does not I will try a fix

**Comment by KaranKad** [comment]
Using legacy installer without developer options, it just prompts me to install or update the app without any errors.

**Comment by Iamlooker** [comment]
So I can try making shizuku a legacy type installer
